src/views/CustomToken.vue

```diff
@@ -136,11 +136,18 @@ export default {
     networkAssets () {
       return this.enabledAssets[this.activeNetwork][this.activeWalletId]
     },
-    isExistingNetworkAsset () {
-      return Boolean(this.networkAssets.find(_symbol => _symbol === this.symbol))
+    chainNativeAsset() {
+      return {
+        ethereum: 'ETH',
```

Comment:
It's a better approach to get chains from cryptoassets.

Reply:
fixed